### PR TITLE
lightning(dm): stop task on data duplication

### PR DESCRIPTION
**Pull request by lance6716**
Signed-off-by: lance6716 <lance6716@gmail.com>

### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflow/issues/3510

#### Tests 

 - Integration test

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Comment by lance6716** [comment]
many changes are replacing pingcap/check to testify 😂

**Review by D3Hunter — APPROVED** [review]
lgtm

**Comment by lance6716** [comment]
/cc @gozssky @buchuitoudegou 

PTAL

File: dm/_utils/terror_gen/errors_release.txt [hunk 1]
```diff
@@ -282,7 +282,8 @@ ErrLoadUnitDuplicateTableFile,[code=34015:class=load-unit:scope=internal:level=h
...
 ErrLoadTaskWorkerNotMatch,[code=34017:class=functional:scope=internal:level=high], "Message: different worker in load stage, previous worker: %s, current worker: %s, Workaround: Please check if the previous worker is online."
 ErrLoadTaskCheckPointNotMatch,[code=34018:class=functional:scope=internal:level=high], "Message: inconsistent checkpoints between loader and target database, Workaround: If you want to redo the whole task, please check that you have not forgotten to add -remove-meta flag for start-task command."
-ErrLoadLightningRuntime,[code=34019:class=load-unit:scope=not-set:level=high]
+ErrLoadLightningRuntime,[code=34019:class=load-unit:scope=internal:level=high]
+ErrLoadLightningHasDup,[code=34020:class=load-unit:scope=internal:level=medium], "Message: physical import finished but the data has duplication, please check `%s`.`%s` to see the duplication., Workaround: You can refer to https://docs.pingcap.com/tidb/stable/tidb-lightning-physical-import-mode-usage#conflict-detection to manually insert data and resume the task."
```
Comment (buchuitoudegou) [inline dm/_utils/terror_gen/errors_release.txt:286]:
```suggestion
ErrLoadLightningHasDup,[code=34020:class=load-unit:scope=internal:level=medium], "Message: physical import finished but the data has duplication, please check `%s`.`%s` to see the duplication, Workaround: You can refer to https://docs.pingcap.com/tidb/stable/tidb-lightning-physical-import-mode-usage#conflict-detection to manually insert data and resume the task."
```
```suggestion
ErrLoadLightningHasDup,[code=34020:class=load-unit:scope=internal:level=medium], "Message: physical import finished but the data has duplication, please check `%s`.`%s` to see the duplication., Workaround: You can refer to https://docs.pingcap.com/tidb/stable/tidb-lightning-physical-import-mode-usage#conflict-detection to manually insert data and resume the task."
```

**Review by buchuitoudegou — APPROVED** [review]

**Pull request merged**